src/components/30-organisms/table-sortable/index.js

```diff
@@ -158,14 +171,43 @@ class AXATableSortable extends LitElement {
       const cleanCellRx = cellRx.replace(/<[^>]*>/g, '').trim();
       const cleanCellLx = cellLx.replace(/<[^>]*>/g, '').trim();
       let result;
-      // eslint-disable-next-line no-restricted-globals
+
+      if (
+        this.dateSortColumnIndex &&
+        this.dateColumsCustomSort.includes(index)
+      ) {
+        const cleanDateLx = new Date(this.convertDateToUSFormat(cleanCellLx));
+        const cleanDateRx = new Date(this.convertDateToUSFormat(cleanCellRx));
+
+        if (!isNaN(cleanDateLx.getTime()) && !isNaN(cleanDateRx.getTime())) {
+          result = sortDate(cleanDateLx, cleanDateRx);
+          return sortAs === ASC ? result : result * -1;
+        }
+      }
+
       if (!isNaN(parseInt(cleanCellLx.charAt(0), 10))) {
         result = this.numCollator.compare(cleanCellLx, cleanCellRx);
       } else {
         result = this.strCollator.compare(cleanCellLx, cleanCellRx);
       }
-      return sortAs === ASC ? result : ~result + 1;
+
+      return sortAs === ASC ? result : result * -1;
     });
+
+    function sortDate(a, b) {
```

Comment:
Why is this a function and the others class methods?

Reply:
To declare it as a class method it has to be outside of the sort() method.

Reply:
This is the default comparator of `sort()`, why do you want to redefine it?

You can sort dates like in this snippet: 
`[+new Date(),+new Date('2018-01-24'),+new Date('2020-10-11')].sort().map(integer => new Date(integer))
(3) [Wed Jan 24 2018 01:00:00 GMT+0100 (Central European Standard Time), Wed Feb 26 2020 13:56:33 GMT+0100 (Central European Standard Time), Sun Oct 11 2020 02:00:00 GMT+0200 (Central European Summer Time)]`

Reply:
If there is an independent reason, I suggest a comment and a better name (it's not a sort, it's a comparator function to be called from a proper sort function).

Reply:
I used an extern comparator, because number and string Collators override default comparator of sort()

Reply:
I don't understand: the sortDate function body seems the same as what the internal comparator would do, hence it would not be needed. Can you disprove that?

Reply:
There are special string and number comparators which override the internal one, so I can't use the internal one, because date, string and number comparison happen in the same sorting function.
`
if (!isNaN(parseInt(cleanCellLx.charAt(0), 10))) {
        result = this.numCollator.compare(cleanCellLx, cleanCellRx);
      } else {
        result = this.strCollator.compare(cleanCellLx, cleanCellRx);
      }
`

Reply:
But now that I use the unix epoch integer it is irrelevant.